.nvmrc

```
@@ -0,0 +1 @@
+v20
```

Let's use the most recent LTS - v20 is in maintenance mode 
```suggestion
v22.14.0
```